examples/basic/main.go

```diff
@@ -7,17 +7,17 @@ import (
 	"net/http"
 	"os"
 	"time"
-
```

Comment:
It's pedantic, but I prefer to keep my stdlib and non-stdlib dependencies separated -- so please revert to having the newline there.